explainer.md

```diff
@@ -46,207 +47,84 @@ This effort aims to create a JavaScript API
-
-### Mapping native HTML to the accessibility tree
-
-Native HTML elements are implicitly mapped to accessibility APIs.
-For example, an  `<img>` element will automatically be mapped
-to an accessibility node with a role of `"image"`
-and a label based on the `alt` attribute (if present).
-
-![<img> node translated into an image on the page and an accessibility node](images/a11y-node-img.png)
-
-
-### ARIA
-
-Alternatively, [ARIA](https://www.w3.org/TR/wai-aria-1.1/)
-allows developers to annotate elements with attributes to override
-the default role and semantic properties of an element -
-but not to expose any accessible actions.
-
-![<div role=checkbox aria-checked=true> translated into a visual presentation and a DOM node](images/a11y-node-ARIA.png)
-
-In either case there's a one-to-one correspondence
-between a DOM node and a node in the accessibility tree,
-and there is minimal fine-grained control over the semantics of the corresponding accessibility node.
-
-## Gaps in the web platform's accessibility story
+(More background on existing APIs can be found in the [Appendices](#appendices).)
 
 Web apps that push the boundaries of what's possible on the web struggle to make them accessible
 because the APIs aren't yet sufficient -
 in particular, they are much less expressive than the native APIs that the browser communicates with.
 
-In the description of each of the phases of the Accessibility Object Model
-spec below, we discuss specific gaps -
-things that currently can't be achieved using current web accessibility APIs,
-or can't be achieved without cumbersome workarounds.
+1. Setting non-reflected (“default”) accessibility properties for [Web Components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) which can be overridden by page authors
+    - Currently, Web Components are forced to use ARIA to declare their default semantics.
+    This causes ARIA attributes which are really implementation details 
+    to "leak" into the DOM. 
+    - This capability _need not_, but _may_ be limited to Web Components.
+2.  Setting [relationship properties](https://www.w3.org/TR/wai-aria-1.1/#attrs_relationships) without needing to use IDREFs
+    - Currently, to specify any ARIA relationship,
+     an author must specify a unique ID on any element which may be the target
+     of the relationship.
+    - In the case of something like 
+    [`aria-activedescendant`](https://www.w3.org/TR/wai-aria-1.1/#aria-activedescendant),
+    this may be one of hundreds or thousands of elements, 
+    depending on the UI.
+    This requirement makes these APIs cumbersome to use
+    and lead to many extra DOM attributes being necessary.
+3. Listening for events from Assistive Technology
+   - Currently, _only_ built-in elements have the capability to react to events,
+     typically triggered by user actions such as 
+     ["click"](https://developer.android.com/reference/android/view/accessibility/AccessibilityEvent.html#TYPE_VIEW_CLICKED) 
```

Comment:
It's not clear to me how this click is different from something like mouse click.

Reply:
Changed to "simulated click".